src/components/ProductInfoBox/ProductInfoBox.js

```diff
@@ -0,0 +1,61 @@
+import React, { useState } from 'react';
+import '../ProductInfoBox/ProductInfoBox.scss';
+import NumBtn from '../ProductInfoBox/NumBtn/NumBtn';
+import Button from '../Button/Button';
+import { useNavigate } from 'react-router';
+
+const ProductInfoBox = ({ productName, productSubTag, originPrice }) => {
+  const [numValue, setNumValue] = useState(1);
+
+  const btnTitle = '바로구매';
+
+  const minusOne = () => {
+    if (numValue === 1) {
+      setNumValue(1);
+    } else {
+      setNumValue(numValue - 1);
+    }
+  };
```

Comment:
```suggestion
  const minusOne = () => {
      setNumValue(numValue === 1 ? 1 : numValue - 1);
```
setNumValue부분이 반복되고 있고 값만 바뀌면 되기에 if else 보다는 삼항연산자 활용하는것이 더 적합해보이네요!